packages/glib.rb

```diff
@@ -33,12 +33,13 @@ class Glib < Package
   depends_on 'util_linux' # R
   depends_on 'zlibpkg' # R
   depends_on 'pcre2' # R
-  no_env_options
+
+  no_strip if ARCH.include['aarch64', 'armv7l']
```

Comment:
Sorry, I meant:
```
no_strip if ['aarch64', 'armv7l'].include?ARCH
```
Don't forget the `?` too.

Reply:
Done!

Reply:
You could probably do:
```
no_strip if %w[aarch64 armv7l].include?ARCH
```